android/src/main/java/com/iterable/iterableapi/RNIterableInternal.java

```diff
@@ -0,0 +1,17 @@
+package com.iterable.iterableapi;
+
+public class RNIterableInternal {
+
+    public static String getEmail(){
+        return IterableApi.getInstance().getEmail();
```

Comment:
This should be using a promise/future somehow I think. Not sure about how this needs to be done for Android.

Reply:
Oh... its not actually related to this ticket. RNIterableInternal is intented to be just a helper class and its method will not be triggered directly by RN javascript side. And hence they will not be returning promise. getEmail() just happened to be the first package private method that I tested and I forgot to remove that. 

Will use this function when adding support to getEmail.

Reply:
removing getEmail to remove ambiguity.. Will add it when adding support to getEmail. :)